packages/web3/scripts/black_box_test.sh

```diff
@@ -30,7 +30,27 @@ echo "RPC client Protocol: " $MODE
 export WEB3_SYSTEM_TEST_PROVIDER="$MODE://localhost:$WEB3_SYSTEM_TEST_PORT"
 export WEB3_SYSTEM_TEST_BACKEND=$BACKEND
 
-cd test/black_box
+cd test/esm_black_box
+yarn --update-checksums
+yarn
+
+if [[ ${BACKEND} == "infura" ]]
+then
+    if [ ! $INFURA_HTTP ] || [ ! $INFURA_WSS ]
```

Comment:
```suggestion
    if [ ! $INFURA_HTTP ] && [ ! $INFURA_WSS ]
```
Maybe we want to exit if both are empty or miss something?

Reply:
switching operands will mean tests will never run